beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -222,83 +236,93 @@ func (s *Service) validateUnaggregatedAttTopic(ctx context.Context, a eth.Att, b
 	return pubsub.ValidationAccept, nil
 }
 
-func (s *Service) validateCommitteeIndexAndCount(
+func (s *Service) validateCommitteeIndex(
```

Comment:
This method has been renamed, but it is still validating the committee count. Want to revert the renaming or refactor further?

Reply:
You're right, I will change it back